addons/website/static/src/interactions/search_modal.js

```diff
@@ -0,0 +1,33 @@
+import { Interaction } from "@website/core/interaction";
+import { registry } from "@web/core/registry";
+
+class SearchModal extends Interaction {
+    static selector = "#o_search_modal_block #o_search_modal";
+    dynamicContent = {
+        _root: {
+            "t-on-show.bs.modal": this.onSearchModalShow,
+            "t-on-shown.bs.modal": this.onSearchModalShown,
+        },
+    }
+
+    /**
+     * @param {Event} ev
+     */
+    onSearchModalShow(ev) {
+        if (!this.editableMode) {
+            return;
+        }
```

Comment:
Maybe highlight this with a TODO because it will have to be handled differently once we integrate with edit mode.